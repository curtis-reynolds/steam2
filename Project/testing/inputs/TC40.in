login
admin
refund
buystd
sellstd
1000000
