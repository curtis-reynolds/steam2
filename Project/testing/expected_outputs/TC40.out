Login successful.
Error: The maximum amount of credit to refund is $999,999.99.
